src/features/empty-base-layer/test/emptyBaseLayer.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('ui.grid.emptyBaseLayer', function () {
+
+  var scope, element, timeout, viewportHeight, emptyBaseLayerContainer;
+
+  beforeEach(module('ui.grid.emptyBaseLayer'));
+
+  beforeEach(inject(function (_$compile_, $rootScope, $timeout, $httpBackend) {
+
+    var $compile = _$compile_;
+    scope = $rootScope;
+    timeout = $timeout;
```

Comment:
Is this timeout variable really needed? It does not get used anywhere. Even $timeout itself is only used inside this beforeEach function.